cloudwatch_logger/src/log_node.cpp

```diff
@@ -30,12 +30,15 @@
 
 using namespace Aws::CloudWatchLogs::Utils;
 
-LogNode::LogNode(int8_t min_log_severity, std::unordered_set<std::string> ignore_nodes) 
-    : ignore_nodes_(std::move(ignore_nodes))
-{
-  this->log_service_ = nullptr;
-  this->min_log_severity_ = min_log_severity;
-}
+LogNode::LogNode(const Options & options)
+  : min_log_severity_(options.min_log_severity),
+    ignore_nodes_(options.ignore_nodes),
+    publish_topic_names_(options.publish_topic_names) {}
+
+LogNode::LogNode(int8_t min_log_severity, std::unordered_set<std::string> ignore_nodes)
+  : min_log_severity_(min_log_severity),
+    ignore_nodes_(std::move(ignore_nodes)),
+    publish_topic_names_(true) {}
```

Comment:
nit: can we instead invoke an options object with `publish_topic_name` set to `true`?

Reply:
Done.